go/logger/standard.go

```diff
@@ -193,8 +193,13 @@ func (log *Standard) Error(fmt string, arg ...interface{}) {
 	}
 }
 
+// Errorf is a deprecated method until all the formatting methods are fixed (use Error instead)
```

Comment:
There is no Debugf or Warningf or Infof... we assume every log has a formatting arg which is why our log statements sometimes contain %MISSING :(